src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -2469,6 +2469,8 @@ private void LoadAnnotation(ExtraAnnotationViewInfo annotationViewInfo)
 
             var text = annotationViewInfo.Title;
 
+            Dictionary<Guid, NodeModel> nodeMap = Nodes.ToDictionary(x => x.GUID);
```

Comment:
Could this be done once before all the Annotations were loaded? Is this just to be sure that new nodes were not loaded between some annotations being loaded?

Reply:
done